apps/src/code-studio/components/AddAssetButtonRow.story.jsx

```diff
@@ -13,40 +13,32 @@ const mockApi = {
   }
 };
 
-export default storybook =>
-  storybook.storiesOf('AddAssetButtonRow', module).addStoryTable([
-    {
-      name: 'Just Buttons',
-      story: () => (
-        <div>
-          <AddAssetButtonRow
-            uploadsEnabled={true}
-            allowedExtensions=""
-            api={mockApi}
-            onUploadStart={() => console.log('onUploadStart')}
-            onUploadDone={() => console.log('onUploadDone')}
-            onUploadError={() => console.log('onUploadError')}
-            onSelectRecord={() => console.log('onSelectRecord')}
-            statusMessage=""
-          />
-        </div>
-      )
-    },
-    {
-      name: 'Buttons and status message',
-      story: () => (
-        <div>
-          <AddAssetButtonRow
-            uploadsEnabled={true}
-            allowedExtensions=""
-            api={mockApi}
-            onUploadStart={() => console.log('onUploadStart')}
-            onUploadDone={() => console.log('onUploadDone')}
-            onUploadError={() => console.log('onUploadError')}
-            onSelectRecord={() => console.log('onSelectRecord')}
-            statusMessage="This is a status message"
-          />
-        </div>
-      )
-    }
-  ]);
+export default {
+  title: 'AddAssetButtonRow',
+  component: AddAssetButtonRow
+};
+
+// Template
+const Template = args => (
+  <AddAssetButtonRow
+    uploadsEnabled={true}
+    allowedExtensions=""
+    api={mockApi}
+    onUploadStart={() => console.log('onUploadStart')}
```

Comment:
same thought as the other PR -- let's use the `action` addon here so the logging is visible in storybook